File: databaseTest2/app/src/main/java/com/example/kuba/databasetest/MainActivity.java
package com.example.kuba.databasetest;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.design.widget.BottomNavigationView;
import android.support.v4.app.Fragment;
import android.support.v7.app.AppCompatActivity;
import android.view.MenuItem;


public class MainActivity extends AppCompatActivity
{

    private BottomNavigationView.OnNavigationItemSelectedListener navListener = new BottomNavigationView.OnNavigationItemSelectedListener()
            {
                @Override
                public boolean onNavigationItemSelected(@NonNull MenuItem item)
                {
                    Fragment selectedFragment = null;

                    switch(item.getItemId())
                    {
                        case R.id.bottom_nav1:
                            selectedFragment = new FragmentEaten();


                            break;
                        case R.id.bottom_nav2:
                            selectedFragment = new FragmentStats();
                            break;
                    }
                    getSupportFragmentManager().beginTransaction().replace(R.id.ma_fragment_container,selectedFragment).commit();
                    return true;
                }
            };

    @Override
    protected void onCreate(Bundle savedInstanceState) {

        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        setContentView(R.layout.activity_main);

        BottomNavigationView bottomNav = findViewById(R.id.ma_bottom_navigation);

        bottomNav.setOnNavigationItemSelectedListener(navListener);
        getSupportFragmentManager().beginTransaction().replace(R.id.ma_fragment_container, new FragmentEaten()).commit();
    }
}

File: databaseTest2/app/src/main/java/com/example/kuba/databasetest/MainActivity.kt
package com.example.kuba.databasetest

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import android.widget.Toast



class MainActivity : AppCompatActivity() {

    lateinit var RegistLink: TextView
    lateinit var LoginButton: Button
    lateinit var LoginLog: EditText
    lateinit var PasswordLog: EditText

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        RegistLink=findViewById(R.id.registLink)
        LoginButton=findViewById(R.id.loginButton)
        LoginLog=findViewById(R.id.loginLog)
        PasswordLog=findViewById(R.id.passwordLog)

        goRegist()
        logIn()
    }

    fun goRegist(){
        RegistLink.setOnClickListener(View.OnClickListener{

            val Intent = Intent(this,RegView::class.java)
            startActivity(Intent);
            return@OnClickListener
        })
    }

    fun logIn (){
        LoginButton.setOnClickListener(View.OnClickListener {
            var userExist: Boolean =false
            val myDb = DatabaseHelper(this)
            val res = myDb.getAllData()

            if(res.getCount()==0){

                return@OnClickListener
            }else{
                while (res.moveToNext()){
                    if(LoginLog.text.toString().trim()==res.getString(1)&& PasswordLog.text.toString().trim()==res.getString(2)){
                        val Intent = Intent(this,addFood::class.java)
                        startActivity(Intent);
                        userExist =true
                    }else{

                    }
                }
                if(!userExist){
                    Toast.makeText(applicationContext,"Wrong login or password", Toast.LENGTH_SHORT).show()
                }


            }
        })
    }
}
